src/actions/AppStoreActions.ts

```diff
@@ -13,15 +14,13 @@ import {
     red,
     teal,
 } from '@material-ui/core/colors';
-import { VariantType } from 'notistack';
-
-import analyticsEnum, { logAnalytics } from '../analytics';
-import { LOAD_DATA_ENDPOINT, SAVE_DATA_ENDPOINT } from '../api/endpoints';
-import { SnackbarPosition } from '../components/AppBar/NotificationSnackbar';
-import { RepeatingCustomEvent } from '../components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog';
-import { CourseDetails, courseNumAsDecimal, getCoursesData, termsInSchedule, warnMultipleTerms } from '../helpers';
-import { Section } from '../peterportal.types';
-import AppStore, { AppStoreCourse, ShortCourseInfo, UserData } from '../stores/AppStore';
+import analyticsEnum, { logAnalytics } from '$lib/analytics';
```

Comment:
Can you resolve the merge conflicts with `main` and re-request review? The diff viewer is showing me these "changes" in the imports that are actually the same as main, and I think it's because it isn't ready to auto-merge.

Reply:
👍